src/django/api/serializers/activity_log.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework.serializers import (
+    ModelSerializer,
+    SerializerMethodField,
+    EmailField,
+    DateTimeField,
+)
+
+from ..models import User, Submission
+
+
+class ActivityDraftedSerializer(ModelSerializer):
+    user = EmailField(source='created_by.email')
+    action = SerializerMethodField()
+    time = DateTimeField(source='created_at')
+
+    def get_action(self, value):
+        return 'Drafted'
+
+    class Meta:
+        model = Submission
+        fields = ['user', 'action', 'time', 'notes']
+        optional_fields = ['annotations_count']
+
+
+class ActivitySubmittedSerializer(ActivityDraftedSerializer):
+    def to_representation(self, instance):
+        obj = super().to_representation(instance)
+        submitter = User.objects.get(pk=instance.submitted_by.pk)
+        obj['action'] = 'Submitted'
+        obj['time'] = instance.submitted_at
+        obj['user'] = submitter.email
+        return obj
+
+
+class ActivityReviewStartedSerializer(ActivityDraftedSerializer):
+    def to_representation(self, instance):
+        obj = super().to_representation(instance)
+        review = instance.review
+        reviewer = User.objects.get(pk=review.reviewed_by.pk)
+        obj['action'] = 'Review Started'
+        obj['time'] = review.created_at
+        obj['user'] = reviewer.email
+        # Don't show review notes in log until review is complete
+        obj['notes'] = ""
+        return obj
+
+
+class ActivityReviewedSerializer(ActivityReviewStartedSerializer):
+    def to_representation(self, instance):
+        obj = super().to_representation(instance)
+        review = instance.review
+        obj['action'] = 'Reviewed'
+        obj['time'] = review.reviewed_at
+        obj['notes'] = review.notes
+        obj['annotations_count'] = review.annotations.count()
+        return obj
+
+
+class ActivityApprovedSerializer(ActivityDraftedSerializer):
```

Comment:
Approvals should not have any notes:

![image](https://user-images.githubusercontent.com/1430060/196276922-228f595a-11d0-4868-a838-c94f7e9bf924.png)